design/ceph/node-loss-rbd-rwo.md

```diff
@@ -0,0 +1,68 @@
+# Recover RBD RWO Pod in case of Node Loss
+
+## Goal
+
+Faster RBD RWO recovery in case of node loss.
```

Comment:
Even though the RWO recovery only specifically haunts RBD, CephFS clients may also cause data
corruption from lost nodes.

@Madhu-1 , I vaguely remember that network fence CR (blocklist cmd) also blocks io for cephfs completely, can you please confirm this ? 

If this is the case, then we can make the design generic both to prevent data corruption and for faster RWO volume recovery in node loss scenario.

Reply:
The CephFS corruption would only happen at the application layer, right? Basically, it's the same as a power loss scenario. While it's true that the blocklist would affect both cephfs and rbd volumes that had been on that node, does rook actually need to blocklist the node if only cephfs volumes were running there?

Reply:
> The CephFS corruption would only happen at the application layer, right? Basically, it's the same as a power loss scenario. While it's true that the blocklist would affect both cephfs and rbd volumes that had been on that node, does rook actually need to blocklist the node if only cephfs volumes were running there?

I think we can keep it generic. The same application pod will be force deleted(k8s etcd database, it may still be running on the  lost data) and started on another data, therefore, there is chances of data corruption for both rbd and cephfs.